core/state/database.go

```diff
@@ -500,18 +500,25 @@ func (db *cachingDB) SetCurrentAccountAddress(addr common.Address) {
 
 func (db *cachingDB) GetCurrentAccountHash() common.Hash {
 	var addrHash common.Hash
-	if db.CurrentTransitionState.CurrentAccountAddress != nil {
+	if db.CurrentTransitionState != nil && db.CurrentTransitionState.CurrentAccountAddress != nil {
```

Comment:
hmmm, the bug is that it is `nil`, this should not happen. I'll deactivate from this PR and will fix it in the merged branch.

Reply:
```suggestion
	if db.CurrentTransitionState.CurrentAccountAddress != nil {
```